ephios/plugins/files/views.py

```diff
@@ -0,0 +1,45 @@
+from django.contrib.messages.views import SuccessMessageMixin
+from django.urls import reverse_lazy
+from django.utils.translation import gettext_lazy as _
+from django.views.generic import CreateView, DeleteView, DetailView, ListView, UpdateView
+from guardian.mixins import LoginRequiredMixin
+
+from ephios.extra.mixins import CustomPermissionRequiredMixin, MediaViewMixin
+from ephios.extra.utils import accelerated_media_response
+from ephios.plugins.files.forms import DocumentForm
+from ephios.plugins.files.models import Document
+
+
+class DocumentView(LoginRequiredMixin, MediaViewMixin, DetailView):
+    model = Document
+
+    def get(self, request, *args, **kwargs):
+        return accelerated_media_response(self.get_object().file)
+
+
+class DocumentListView(CustomPermissionRequiredMixin, ListView):
+    model = Document
+    permission_required = "files.add_document"
```

Comment:
Needs a way to acquire these permissions. 
To that regard I think every plain responsible should be able to upload files for their event but not be able to delete files from other users?!

Reply:
I added a setting to the group form before your comment. I'm not sure if we should tie file permissions to events so closely. The reason to keep the files separate in the first place was the flexibility to also use them somewhere else in ephios, e.g. on a page to have a Mitgliedsantrag ready at hand or similar

Reply:
I do think it makes sense to have some general file storage, however

* This is missing a non-event related way to access files currently, right?
* I think event-specific files would clutter a list with some general purpose files. Maybe file organization needs some more thought...